consensus/state.go

```diff
@@ -67,6 +67,7 @@ type Network struct {
 		FixHeight        uint64    `json:"fixHeight"`
 		GenesisTimestamp time.Time `json:"genesisTimestamp"`
 	} `json:"hardforkOak"`
+	//nolint:tagliatelle
```

Comment:
unfortunate. There's an issue open for specifying custom initialisms in tagliatelle, but no indication that it'll be implemented any time soon.

Reply:
btw, old tag name needs to be restored

Reply:
Sorry about that... Didn't sleep well last night